deploy/Dockerfile.arm64

```diff
@@ -0,0 +1,10 @@
+FROM multiarch/alpine:aarch64-edge
```

Comment:
Wouldn't it be more consistent to use `arm64v8/alpine:3.7` here so this matches `Dockerfile.amd64`?

(If you visit https://hub.docker.com/_/alpine/ and click on `arm64v8` under "Supported architectures:", it takes you to https://hub.docker.com/r/arm64v8/alpine/, which is the architecture-specific push location for official images and shows that 3.7 is supported there.)